packages/protocol/contracts/identity/Attestations.sol

```diff
@@ -249,7 +306,10 @@ contract Attestations is
     );
 
     // solhint-disable-next-line not-rely-on-time
-    require(isAttestationTimeValid(attestation.time), "Attestation request timed out");
+    require(
+      isAttestationRequestBlockTimeValid(attestation.blockNumber),
+      "Attestation request timed out"
```

Comment:
Is it the *request* that timed out here?

Reply:
Hmm, yeah I just realized that we have two different notions of "request".

First is the request where you commit the attestation fee and one has to call `selectIssuers` for to set the attestation.

Second is the request for that attestation to be completed to the issuer, either by calling the `reveal` tx or making a request to the attestation service.

Both represent the notion of a user requesting attestations, albeit at different abstractions.

I'll change it so that only the former is named `request`, the latter being just `attestation`, but lmk if that is also confusing